### PR TITLE
Fix files not added to root of archive

### PR DESCRIPTION
**Pull request by phavekes**
I'm not shure why '*' was excluded, but it caused the new component_info was missing in the archive, and deploy failing.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -87,7 +87,6 @@
     },
     "archive": {
         "exclude": [
-            "*",
```
Comment (pablothedude) [inline composer.json:90]: The trick is to first exclude all files and only add the files needed. That way we get a clean tarball with only the required files and no testing debugging or other files on production we don't want. That way the tarball is as lightweight as possible. Probably the best thing to do is  to add `!/component_info` and revert this change.

**Review by pablothedude — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (phavekes) [reply composer.json:90]: Thank you for your suggestion. I've made the suggested updates.
